### PR TITLE
Update azure-pipeline 

### PR DESCRIPTION
**Pull request by RupinderKaurSSB**
Updated configuration for azure-pipeline.
1. Added ref to branch in template repo:  "sonarQube-to-sonarCloud"
2. Variable for sonarCloud: "bipSonarCloud-Dapla"
3. Updated sonar-project.properties with projectKey

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,8 +39,7 @@
     "eject": "react-scripts eject",
     "serve": "serve -s build -l 3000",
     "stacker-postinstall": "npm run build",
-    "prestart": "react-env --dest ./public",
-    "coverage": "react-scripts test --coverage --watchAll=false"
+    "coverage": "react-scripts test --coverage --watchAll=false --passWithNoTests "
```
Comment (mmj-ssb) [inline package.json:42]: Remove trailing space

File: package.json [hunk 2]
```diff
@@ -70,28 +69,5 @@
...
-        "statements": 75
-      }
-    },
-    "coverageReporters": [
-      "cobertura",
-      "lcov",
-      "text"
-    ]
-  }
+  ]
```
Comment (mmj-ssb) [inline package.json:72]: You should keep collectCoverageFrom in some form and coverageReporters. It is only necessary to remove coverageThreshold.

File: sonar-project.properties [hunk 3]
```diff
@@ -1,8 +1,9 @@
...
-sonar.cpd.exclusions=**/node_modules/**, **/build/**, **/coverage/**, **/enums/**, **/configurations/**, **/__tests__/**
-sonar.coverage.exclusions=**/node_modules/**, **/build/**, **/coverage/**, **/enums/**, **/configurations/**, **/__tests__/**, **/index.js, **/setupTests.js
-sonar.javascript.lcov.reportPaths=./coverage/lcov.info
+sonar.sources=.
+sonar.exclusions=**/node_modules/**, **/build/**, **/coverage/**, **/enums/**
+sonar.test.exclusions=**/node_modules/**, **/build/**, **/coverage/**, **/enums/**
+sonar.javascript.exclusions=**/node_modules/**, **/build/**, **/coverage/**, **/enums/**
+sonar.cpd.exclusions=**/node_modules/**, **/build/**, **/coverage/**, **/enums/**, **/__tests__/**
+sonar.coverage.exclusions=**/node_modules/**, **/build/**, **/coverage/**, **/enums/**, **/__tests__/**, **/index.js, **/setupTests.js
+sonar.javascript.lcov.reportPaths=./coverage/lcov.info
```
Comment (mmj-ssb) [inline sonar-project.properties:9]: Newline

Comment (RupinderKaurSSB) [reply sonar-project.properties]: Done.

File: package.json (hunk 2 above)
Comment (RupinderKaurSSB) [reply package.json]: Done

File: package.json (hunk 1 above)
Comment (RupinderKaurSSB) [reply package.json]: Done

**Review by mmj-ssb — APPROVED** [review]
Perfect! 👍

**Pull request merged**